### PR TITLE
feat(rbac): show total no. of rules in review and create step

### PR DESCRIPTION
**Pull request by divyanshiGupta**
Story: https://issues.redhat.com/browse/RHIDP-2771

Screenshot
Screenshot 2024-06-20 at 4 59 43 PM

### CODE REVIEW

**Comment by divyanshiGupta** [comment]
cc: @ShiranHi

**Comment by ShiranHi** [comment]
Looks great @divyanshiGupta !

File: plugins/rbac/src/components/CreateRole/ReviewStepTable.tsx [hunk 1]
```diff
@@ -3,17 +3,25 @@ import React from 'react';
...
   rows: any[];
+  tableWrapperWidth: number;
 }) => {
   return (
-    <div style={{ maxHeight: '230px', overflow: 'scroll', width: '650px' }}>
-      <table style={{ width: '600px' }}>
+    <div
+      style={{
+        maxHeight: '230px',
+        overflow: 'scroll',
```
Comment (invincibleJai) [inline plugins/rbac/src/components/CreateRole/ReviewStepTable.tsx:16]:
```suggestion
        overflow: 'auto',
```

should this be auto?

**Pull request merged**